logging/cluster-logging-release-notes.adoc

```diff
@@ -8,34 +8,65 @@ toc::[]
 [id="openshift-logging-about-this-release"]
 == About this release
 
-The following advisories are available for Red Hat OpenShift Logging:
+The following advisories are available for Red Hat OpenShift Logging versions 5.1.x and 5.2.x:
 
+* link:https://access.redhat.com/errata/RHBA-2021:2885[RHBA-2021:2885 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.1]
 * link:https://access.redhat.com/errata/RHBA-2021:2112[RHBA-2021:2112 - Bug Fix Advisory. OpenShift Logging Bug Fix Release 5.1.0]
-* link:https://access.redhat.com/errata/RHBA-2021:2655[RHBA-2021:2655 - Bug Fix Advisory. Openshift Logging Bug Fix Release (5.0.6)]
-* link:https://access.redhat.com/errata/RHSA-2021:2374[RHSA-2021:2374 - Security Advisory. Moderate: Openshift Logging Bug Fix Release (5.0.5)]
-* link:https://access.redhat.com/errata/RHSA-2021:2136[RHSA-2021:2136 - Security Advisory. Moderate: Openshift Logging security and bugs update (5.0.4)]
-* link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 - Security Advisory. Important OpenShift Logging Bug Fix Release (5.0.3)]
-* link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.2)]
-* link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.1)]
-* link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 - Bug Fix Advisory. Errata Advisory for Openshift Logging 5.0.0]
 
 [id="openshift-logging-supported-versions"]
 == Supported versions
 
-* OpenShift Logging version 5.0 runs on {product-title} versions 4.7 and 4.8.
-* OpenShift Logging version 5.1 runs on {product-title} versions 4.7 and 4.8.
+* OpenShift Logging versions 5.1 and 5.2 run on {product-title} versions 4.7 and 4.8.
```

Comment:
In the description of this PR it says the change applies for 4.8+, so I just wanted to check that this is accurate?

Reply:
Yes. This information is correct and appropriate for the OpenShift Logging 5.2 docs, which lands in enterprise-4.8.
The OpenShift Logging 5.1 docs, which landed in enterprise-4.7, contain a similar statement. https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-release-notes.html